EntAppProject/src/test/java/com/jobook/EntAppProject/JobServiceUnitTests.java

```diff
@@ -1,93 +1,60 @@
 package com.jobook.EntAppProject;
 
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-
 import java.util.List;
-
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
-
 import com.jobook.EntAppProject.Services.IJobService;
 import com.jobook.EntAppProject.Services.IUserService;
-import com.jobook.EntAppProject.model.Customer;
-import com.jobook.EntAppProject.model.Employee;
 import com.jobook.EntAppProject.model.Job;
 import com.jobook.EntAppProject.model.User;
 
+import static org.junit.Assert.*;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class JobServiceUnitTests {
 	@Autowired
 	IJobService jobService;
 	@Autowired
 	IUserService userService;
-	
-	User user;
-	List<Job> jobs;

```

Comment:
Doing this makes debugging annoying. While it works, it makes more sense to scope each variable to its respective test. That makes it easier when someone else comes in to help, and it also reducing the risk of concurrency issues (Even though it is a test).